slack_sdk/web/client.py

```diff
@@ -119,32 +114,114 @@ def admin_apps_approve(
                 "The app_id or request_id argument must be specified."
             )
 
-        return self.api_call("admin.apps.approve", json=kwargs)
+        kwargs.update(
+            {
+                "enterprise_id": enterprise_id,
+                "team_id": team_id,
+            }
+        )
+        return self.api_call("admin.apps.approve", params=kwargs)
```

Comment:
As some endpoints do not support `application/json` request body, it's safer to use the form body for most APIs.

Reply:
Interesting, but all endpoints support form POSTs?

Reply:
@filmaj Yes, they do but interestingly some endpoints do not support application/json.